src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -157,7 +157,7 @@
 	<PackageReference Include="SharpDX.Direct3D9" Version="4.2.0" />
 	<PackageReference Include="SharpDX.DXGI" Version="4.2.0" />
 	<PackageReference Include="SharpDX.Mathematics" Version="4.2.0" />
-	<PackageReference Include="System.Reflection.MetadataLoadContext" Version="6.0.0" />
+	<PackageReference Include="System.Reflection.MetadataLoadContext" Version="8.0.0" />
```

Comment:
why bump up the version ?
ANy issues compatibility issues with hosts ? Need to update licensing ?
Will it cause issues if we cherry pick this in 2.19.5 ?

Reply:
1. why use an old version?
2. I don't think so, I can look at Revit, but I think aligning on the .net8 version makes the most sense and avoids conflicts with other packages, like meta data load context net8.
3. I don't think we should cherry pick this.

Reply:
@pinzart90 WRT to licensing were you steps for regenerating the license files documented somewhere in the release process notes - I can't find them atm.

Reply:
Regarding the license, I was thinking we shoul just update the about box stuff. The dependency reports can be postponed